scripts/repo_data.sh

```diff
@@ -36,6 +36,10 @@ resolve_app_version() {
         set_var APP_RELEASE_TYPE "Beta (Pre-release)"
         set_var APP_VERSION "${GITHUB_REF##refs/heads/release/}-rc.${GITHUB_RUN_NUMBER}"
       fi
+      if [[ -z "${GITHUB_REF##*/poc/*}" ]]; then
+        set_var APP_RELEASE_TYPE "Experimental"
```

Comment:
Align RELEASE_TYPE with the acronym=> e.g. "Proof of Concept" or "Proof of Concept (Experiment)"